memory/migrate_pages.py

```diff
@@ -50,16 +50,23 @@ def setUp(self):
             self.cancel('Test requires two numa nodes to run.'
                         'Node list with memory: %s' % nodes)
 
-        dist = distro.detect()
+        self.dist = distro.detect()
         pkgs = ['gcc', 'make']
-        if dist.name in ["Ubuntu", 'debian']:
+        if self.dist.name in ["Ubuntu", 'debian']:
             pkgs.extend(['libpthread-stubs0-dev',
                          'libnuma-dev', 'libhugetlbfs-dev'])
-        elif dist.name in ["centos", "rhel", "fedora"]:
-            pkgs.extend(['numactl-devel', 'libhugetlbfs-devel'])
-        elif dist.name == "SuSE":
+        elif self.dist.name in ["centos", "rhel", "fedora"]:
+            if (self.dist.name == 'rhel' and self.dist.version == '9'):
+                self.log.info("libhugetlbfs is not available RHEL 9.x onwards,\
+                                so tests related to hugepage will be cancelled")
+                pkgs.extend(['numactl-devel'])
```

Comment:
Move this outside of RHEL 9 case.

Reply:
Sure, so we would only have  "numactl-devel" package for all centos, fedora and rhel-9 , I  believe !

Reply:
> Sure, so we would only have "numactl-devel" package for all the three, centos, fedora and rhel-9 , I believe !

No. numactl for all centos, fedora and RHEL (including 9)
libhugetlbfs for all centos, fedora and RHEL (excluding 9)
 
if (RHEL, Fedora, CentOS)
     packages.extend(numactl-devel)
     if (!rhel9)
           packages.extend(libhugetlbfs-devel)